architecture-decision-record/0028-expand-the-scope-of-the-platform.md

```diff
@@ -0,0 +1,77 @@
+# 28. Expand the Scope of the Platform
+
+Date: 2023-11-14
+
+## Status
+
+✅ Accepted
+
+## Context
+
+The Modernisation Platform was originally developed with the intention that it would only be used for migrating legacy applications to a strategic cloud platform, where they could not be hosted in the [Cloud Platform](https://github.com/ministryofjustice/cloud-platform).  Applications would then be hosted on the Modernisation Platform until they were either modernised and moved to the Cloud Platform, or decommissioned or replaced with newer applications.
+
+As the platform has developed and grown, we have encountered scenarios where new applications are required, but the Cloud Platform is not able to host them. Examples of this include other hosting platforms, and applications using AWS services which have certain technical requirements which cannot run on the Cloud Platform, and there are no Cloud Platform compatible alternatives.  There is also a greater understanding now that some applications which were thought to be decommissioned are likely to remain for many years.
+
+## Options
+
+### 1. Create another platform for applications which can't be hosted on the Cloud Platform
+
+**Pros**
+
+- Dedicated platform for non container related workflows
+
+**Cons**
+
+- Will likely be extremely similar to the Modernisation Platform
+- New team to build required and maintain
+- Expensive
+
+### 2. Only allow applications which can be hosted on the Cloud Platform
+
+**Pros**
+
+- Consistent infrastructure throughout the MoJ
+- No additional work required
+- No need for an additional platform
+
+**Cons**
+
+- Forces teams to use technologies which may not fit the application
+- Still have long term legacy applications requiring the Modernisation Platform
+- Places more pressure on the Cloud Platform to host different technologies
+
+### 3. Give AWS accounts directly to applications
+
+**Pros**
+
+- Well it's an option
+
+**Cons**
+
+- Goes against hosting strategy
+- Teams responsible for building entire AWS infrastructure, landing zones etc
+- A step backwards towards the cloud wild west
+
+### 4. Applications which can be hosted on the Cloud Platform must be, but if there is a valid reason why they can't, then applications or platforms can be hosted on the Modernisation Platform
```

Comment:
Maybe reframe this as “services” instead of applications, and make it clear above that when you say services, you mean both user-facing apps _and_ platforms?

Reply:
Thanks I prefer this, I've replaced throughout